vqpy/utils/filters.py

```diff
@@ -0,0 +1,34 @@
+class continuing:
+    """A filter constraint that checks if the condition on property
+    is True for a certain duration
+
+    Returns True if the condition on `property` is True for the given
+    duration, with duration being counted accumulatively.
+    Attribute with name `property`_duration is used to store the
+    cumulative duration of condition being met.
```

Comment:
Shall we change the meaning of `property_duration` to a time range that the condition is satisfied?
Like the time periods that loitering occurs. value of (10, 20) means loitering occurs from 10s to 20s in the video; [(10, 20), (25, 60)] means that there are two time periods when the same person loiters is detected, one is from 10s to 20s, another one is from 25s to 60s. Note that both time periods should exceed the duration user specified

Reply:
Changed as specified.